package/libgluonutil/src/libgluonutil.c

```diff
@@ -109,6 +109,31 @@ struct json_object * gluonutil_wrap_and_free_string(char *str) {
 	return ret;
 }
 
+
+bool gluonutil_get_mesh_prefix(struct in6_addr *result) {
+	struct json_object *site = gluonutil_load_site_config();
+	if (!site)
+		return false;
+
+	struct json_object *node_prefix = NULL;
+	if (!json_object_object_get_ex(site, "node_prefix6", &node_prefix))
+		return false;
+
+	char *str_prefix = (char*)json_object_get_string(node_prefix);
```

Comment:
This `(char*)` cast doesn't make sense, just make the variable a `const char *`.

Reply:
done.